# start the turtlebot to follow the waypoint
---